os-kieserver-launch/added/kieserver-config.sh

```diff
@@ -155,8 +155,13 @@ function setKieServerEnv() {
             JBOSS_NODE_NAME="${HOSTNAME}"
         fi
         if [ "x${JBOSS_NODE_NAME}" != "x" ]; then
+            # [CLOUD-2105] - Process server does not truncate the node name to 23 bytes
+            if [ ${#JBOSS_NODE_NAME} -gt 23 ]; then
+                JBOSS_NODE_NAME=${JBOSS_NODE_NAME: -23}
```

Comment:
This will not allow the recovery of transactions if the deployment scales down as the node name will never be the same again